nbs/01_hello_world.ipynb

```diff
@@ -6,75 +6,14 @@
    "metadata": {},
```

Comment:
_Commented on notebook `nbs/01_hello_world.ipynb` Cell 4 Line 2_
```markdown
# Additional Learning Resources
Make sure you check out [Code Standards](https://dksund.sharepoint.com/:fl:/g/contentstorage/CSP_7c761ee7-b577-4e08-8517-bc82392bf65e/EVddgBNVhOxJm8q27-KyX-UBdfPBbcqtjMK1DquKhDUPiA?e=E8uw7f&nav=cz0lMkZjb250ZW50c3RvcmFnZSUyRkNTUF83Yzc2MWVlNy1iNTc3LTRlMDgtODUxNy1iYzgyMzkyYmY2NWUmZD1iJTIxNXg1MmZIZTFDRTZGRjd5Q09TdjJYblkwVlNiWXFYcE1yaHVrVmZqTVJUVEE4X1VwZjhTd1JxcjRNdmFrSmh2RCZmPTAxVlVLVzVWS1hMV0FCR1ZNRTVSRVpYU1ZXNTdSTEVYN0YmYz0lMkYmYT1Mb29wQXBwJnA9JTQwZmx1aWR4JTJGbG9vcC1wYWdlLWNvbnRhaW5lciZ4PSU3QiUyMnclMjIlM0ElMjJUMFJUVUh4a2EzTjFibVF1YzJoaGNtVndiMmx1ZEM1amIyMThZaUUxZURVeVpraGxNVU5GTmtaR04zbERUMU4yTWxodVdUQldVMkpaY1Zod1RYSm9kV3RXWm1wTlVsUlVRVGhmVlhCbU9GTjNVbkZ5TkUxMllXdEthSFpFZkRBeFZsVkxWelZXU1RJMVJsaFBNalkyUlZkQ1FqTTFRVmhKVTBkRFVVcFdXa1klM0QlMjIlMkMlMjJpJTIyJTNBJTIyNGQ4MWUyZGQtMTUyMy00Y2U1LTg3OWItZWU1ZTNhYWUyOThiJTIyJTdE) (<b>Recommended</b> requires SSI login)
```
Not sure if stripping out the references to information is the best idea. I know we have the loop but this of course is a link to it.

Reply:
Im fine with keeping links to resources